### PR TITLE
Removes nullcrates

### PR DESCRIPTION
**Pull request by bloons3**
### About The Pull Request
deletes nullcrates

### Why It's Good For The Game
people will always find a way to mass-produce nullcrates
the only reason people play cargo is for a chance to get null crates, and if you're only playing a role to get antag loot that's the same as self-antagging without the license to kill
it encourages people to do shitty things with loot that is almost exclusively designed to kill other people
economy and the money system is rarely balanced
when so much of the loot table is a problem it's probably a better idea to cut it outright than to curate the list
you can exponentially increase the value of your TC by simply buying an emag and a bunch of briefcases

Trust me.
## Changelog
:cl: plapatin
del: deletes null crates
/:cl:

### CODE REVIEW

**Comment by MNarath1** [comment]
Very funny
"author: "plapatin"
delete-after: True
changes: 
  - rscdel: "deletes null crates""

**Comment by MNarath1** [comment]
Also liking your own PR really bloons?

**Review by ghost — DISMISSED** [review]

**Comment by Sinmero** [comment]
> Trust me.

bruh

**Comment by TeomanTheGreat** [comment]
No changelog. Smh

**Comment by OceanFish1** [comment]
another null crate nerf moment

**Review by francinum — APPROVED** [review]

**Comment by bloons3** [comment]
image

**Review by francinum — CHANGES_REQUESTED** [review]
Give it a changelog, you know the rules.

**Comment by park66665** [comment]
`/datum/supply_pack/emergency/specialops` has a reference to `NULL_ENTRY`. You probably want to change that too.

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -279,7 +242,7 @@
 
 /datum/supply_pack/emergency/specialops
 	name = "Special Ops Supplies"
-	desc = "(*!&@#TOO CHEAP FOR THAT NULL_ENTRY, HUH OPERATIVE? WELL, THIS LITTLE ORDER CAN STILL HELP YOU OUT IN A PINCH. CONTAINS A BOX OF FIVE EMP GRENADES, THREE SMOKEBOMBS, AN INCENDIARY GRENADE, AND A \"SLEEPY PEN\" FULL OF NICE TOXINS!#@*$"
+	"(*!&@#SAD ABOUT THAT NULL_ENTRY, HUH OPERATIVE? WELL, THIS LITTLE ORDER CAN STILL HELP YOU OUT IN A PINCH. CONTAINS A BOX OF FIVE EMP GRENADES, THREE SMOKEBOMBS, AN INCENDIARY GRENADE, AND A \"SLEEPY PEN\" FULL OF NICE TOXINS!#@*$"
```
Comment (park66665) [inline code/modules/cargo/packs.dm]:
```suggestion
	desc = "(*!&@#SAD ABOUT THAT NULL_ENTRY, HUH OPERATIVE? WELL, THIS LITTLE ORDER CAN STILL HELP YOU OUT IN A PINCH. CONTAINS A BOX OF FIVE EMP GRENADES, THREE SMOKEBOMBS, AN INCENDIARY GRENADE, AND A \"SLEEPY PEN\" FULL OF NICE TOXINS!#@*$"
```

Comment (bloons3) [reply code/modules/cargo/packs.dm]: 😰

**Comment by PowerfulBacon** [comment]
🔑

**Comment by BowStache** [comment]
only reason to play cargo smh

**Comment by bloons3** [comment]
> only reason to play cargo smh

If the only reason you play Cargo is for the chance of antag gamer gear, play on a different server.

**Comment by mudzbe** [comment]
> only reason to play cargo smh

THIS and guns. This PR is massive cringe

> > only reason to play cargo smh
> 
> If the only reason you play Cargo is for the chance of antag gamer gear, play on a different server.

He does

**Comment by Edgar-Allan-ShitPoest** [comment]
I gotta say this is one of the dumbest PR ideas. Maybe instead of removing nullcrates you should work on something else cargo can do?

**Comment by BowStache** [comment]
bloons3? more like cringe3

**Comment by Sinestia** [comment]
> 
> 
> I gotta say this is one of the dumbest PR ideas. Maybe instead of removing nullcrates you should work on something else cargo can do?

You're welcome to do so, pretty much everyone agrees cargos in a pretty poor state 

That's not an argument against the removal of nulls tho lol

If I had to guess the only alternative that might be accepted would be like capping the purchases to 1 (but i doubt that tbh)

**Comment by Edgar-Allan-ShitPoest** [comment]
> 
> 
> > I gotta say this is one of the dumbest PR ideas. Maybe instead of removing nullcrates you should work on something else cargo can do?
> 
> You're welcome to do so, pretty much everyone agrees cargos in a pretty poor state
> 
> That's not an argument against the removal of nulls tho lol
> 
> If I had to guess the only alternative that might be accepted would be like capping the purchases to 1 (but i doubt that tbh)

Agree with sucking my nuts lole :^).
Stop nerfing fun shit because you wanna be a giant baby.

**Comment by Sinestia** [comment]
> 
> 
> > > I gotta say this is one of the dumbest PR ideas. Maybe instead of removing nullcrates you should work on something else cargo can do?
> > 
> > 
> > You're welcome to do so, pretty much everyone agrees cargos in a pretty poor state
> > That's not an argument against the removal of nulls tho lol
> > If I had to guess the only alternative that might be accepted would be like capping the purchases to 1 (but i doubt that tbh)
> 
> Agree with sucking my nuts lole :^).
> Stop nerfing fun shit because you wanna be a giant baby.

https://www.youtube.com/watch?v=NBLyl2UIoVs

**Comment by AtlasRules** [comment]
griefsky is already EMP proof, unless you have 2/3 people or a borg you're not going to live long unless you're not unrobust

terrible PR/10 too much /tg/

**Comment by ghost** [comment]
> Agree with sucking my nuts lole :^).
> Stop nerfing fun shit because you wanna be a giant baby.

Hello moccha.

**Comment by mudzbe** [comment]
Bloons, take notes https://github.com/BeeStation/BeeStation-Hornet/pull/3723 this is how a real man does a PR

**Comment by Edgar-Allan-ShitPoest** [comment]
> 
> 
> > Agree with sucking my nuts lole :^).
> > Stop nerfing fun shit because you wanna be a giant baby.
> 
> Hello moccha.

i dont know who moccha is

**Comment by mudzbe** [comment]
YOU CANNOT CENSOR THE TRUTH

**Comment by park66665** [comment]
Headcoder will be informed of this situation as soon as possible. Until @ike709 makes decision, discussion for this PR will be locked.

**Comment by jupyterkat** [comment]
everything's fine
image

**Comment by ghost** [comment]
This is fine

**Review by bloons3 — COMMENTED** [review]
added changelog

**Review by bloons3 — COMMENTED** [review]
resolve fran's requested change

**Review by francinum — APPROVED** [review]
Approved without question. I have almost never seen these used by antags without the direct and active aid of cargo.

**Comment by ike709** [comment]
At first I wasn't a major fan of this. Then I was shown the logs of how often these get ordered in a round. It's literally multiple dozens per round whenever Cargo gets the credits.

I'm invoking "we can't have nice things".

**Pull request merged**